app/views/articles/_full_comment_area.html.erb

```diff
@@ -16,6 +17,8 @@
         data-commentable-id="<%= @article.id %>"
         data-commentable-type="Article"
         data-has-recent-comment-activity="<%= @article.has_recent_comment_activity? %>">
+
+        <%= render "/comments/discussion_lock_reason" %>
```

Comment:
We display the markup regardless and use JS to hide/show accordingly because we can't rely on the markup being updated fast enough since we bust the edge cache asynchronously.